.circleci/config.yml

```diff
@@ -92,6 +92,15 @@ jobs:
 
     steps:
       - checkout
+      - run: 
+          name: download Go 1.14.13
```

Comment:
This is for the Windows build, which otherwise defaults to 1.13. Our minimum supported version is 1.14, and 1.13 happened to work OK for a while, but it stopped working when we had to temporarily reference a private branch.